docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_constructions.md

```diff
@@ -0,0 +1,103 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_constructions.md"]
+```
+
+# Constructions
+
+The standard way to define a polyhedron is by either giving a V-description or an H-description.
+But polyhedra may also be constructed through other means: by name, via operations on other polyhedra, or from other objects in Oscar.
+
+## H and V representations
+
+### Intersecting halfspaces: H-description
+
+Based on the definition of an $H$-representation, the constructor of `Polyhedron` can be used:
+
+```@docs
+Polyhedron(A::Union{Oscar.MatElem,AbstractMatrix}, b)
+```
+
+The following defines a polytope, as a `Polyhedron`, via a $V$-representation by calling the following function:
+
+```@docs
+convex_hull(::AnyVecOrMat; non_redundant::Bool=false)
+```
+
+### Computing convex hulls: V-description
+
+This is a standard triangle, defined via a (redundant) $V$-representation  and its unique minimal $H$-representation:
+
+```@repl oscar
+T = convex_hull([ 0 0 ; 1 0 ; 0 1; 0 1/2 ])
+facets_as_halfspace_matrix_pair(T)
+```
+
+
+
+
+## Named polyhedra
+Some commonly used polyhedra are able to be constructed by name. For example
```

Comment:
I don't think polyhedra "are able" to construct anything. Also, this is a reference manual, so why "for example"? Don't you list all constructors we provide here? If not, why not?

Personally, I'd just drop this sentence completely (like many of the others, although I won't repeat this each time). But if you really want to keep it, how about this:
```suggestion
For some commonly used types of polyhedra, dedicated constructors are provided:
```